src/App.tsx

```diff
@@ -1,7 +1,27 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+export const App: React.FC = () => {
+  const [key, setKey] = useState('');
+
+  const handleKeyPress = (event: KeyboardEvent) => {
+    setKey(event.key);
+  };
+
+  useEffect(() => {
+    document.addEventListener('keyup', handleKeyPress);
+
+    return () => {
+      document.removeEventListener('keyup', handleKeyPress);
+    };
+  }, [key]);
```

Comment:
![image](https://github.com/mate-academy/react_keyboard/assets/91376823/bbe2a8c4-c467-4390-843b-1a6384677180)
but you used componentDidUpdate

Reply:
@DenisChakhalian good point, forgot to fix it, thanks!